styles.css

```diff
@@ -2,20 +2,33 @@ body {
   background-color: #e5e5e5;
```

Comment:
- Kindly ensure that your project sticks as much as possible to the Figma template provided. There is a white margin around your project that doesn't exist in the Figma template. :eyes: :eyes: Kindly remove it. 

Your design
![image](https://user-images.githubusercontent.com/98363075/199860459-67baa13d-c9f1-4cf4-bcb6-a9a63a5bea33.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/199860565-ae81c009-7484-4c6f-9939-9aa961292aa1.png)

- Please make sure that your project is rendering all the content without needing horizontal scrolling. It improves the user's experience and will make your project shows more professional. :+1: :smiley: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/199860995-97397680-37a1-4803-abd0-364c2b710643.png)